howdoi/howdoi.py

```diff
@@ -453,6 +453,11 @@ def howdoi(raw_query):
         parser = get_parser()
         args = vars(parser.parse_args(raw_query.split(' ')))
 
+    if args.get('plugin'):
```

Comment:
Maybe we might want to keep the plugin parsing _after_ the optional cache clearance code, because if the user decided to clean the cache then the program would clear the cache and stop (we're not interested in parsing the plugin).